conda-forge.yml

```
@@ -4,3 +4,7 @@ github:
 conda_build:
```

To free up disk space on Azure CI runners, would use [the `azure` config section]( https://conda-forge.org/docs/maintainer/conda_forge_yml/#azure ) and add `free_disk_space` as `true` like below

```suggestion
azure:
  free_disk_space: true
conda_build:
```

---

Thanks ! Will further arrange the order of keys, and update `about` section